suite-native/assets/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const MAX_ASSETS_ON_DASHBOARD = 3;
```

Comment:
I would say this should be in module-home since it's tied to dashboard and components that use it shoud accept it as prop

Reply:
fixup 58e280ba4c